h5p-bildetema/src/components/CollectionsController/MultiLanguageWord/MultiLanguageWord.tsx

```diff
@@ -85,25 +86,18 @@ export const MultiLanguageWord = ({
       <div>
         {isCollectionOwner ? (
           <Menu>
-            <Menu.Button className={styles.button}>
-              <MoreVertIcon />
-            </Menu.Button>
-            <Menu.Items className={styles.menu}>
-              <Menu.Item>
-                {({ active }) => (
-                  <button
-                    type="button"
-                    className={`${styles.menuItemButton} ${
-                      active && styles.active
-                    }`}
-                    onClick={() => setOpenDialog(OpenDialog.DELETE_DIALOG)}
-                  >
-                    <DeleteIcon />
-                    <span>Slett</span>
-                  </button>
-                )}
-              </Menu.Item>
-            </Menu.Items>
+            <MenuButton className={styles.menuButton}>
+              <Button variant="circle">
+                <MoreVertIcon />
+              </Button>
+            </MenuButton>
+            <MenuItems anchor="bottom end">
+              <MenuItem
+                label="Slett"
```

Comment:
The label text must be translated later, add a TODO for it

Reply:
Done